src/background/Wallet/Wallet.ts

```diff
@@ -1713,19 +1750,32 @@ export class Wallet {
   }
 
   private async checkTestnetMode(
-    chainId: ChainId
+    options:
+      | { chainId: ChainId; id?: undefined }
```

Comment:
Can you use your new cool helper here? :)

Reply:
I actually didn't add it to our codebase yet because it relies on `'type-fest'` dependency, which I'm not yet sure we need :)